src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
+function Login() {
```

Comment:
**suggestion**: Consider adding PropTypes or TypeScript interfaces to define expected props for better type safety.